src/world_interface/real_world_interface.cpp

```diff
@@ -67,13 +67,35 @@ void initMotors() {
 	can::motor::initMotor(motorSerialIDMap.at(motorid_t::rearLeftWheel));
 	can::motor::initMotor(motorSerialIDMap.at(motorid_t::rearRightWheel));
 
+	// loop over potMotors and init accordingly with can::motor::initPotentiometer
+	// loop over encMMotors and init accordingly
```

Comment:
These comments are a bit superfluous, we can remove them, since they just describe a for loop.